src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -194,24 +207,39 @@ public async Task<IReadOnlyList<SearchServiceProperties>> GetSearchServicesAsync
             return searchServices;
         }
 
-        private async Task<SearchServiceProperties> GetSearchServiceFromAzureAsync(AzureCloudServiceDetails serviceDetails, ITestOutputHelper logger)
+        private async Task<SearchServiceProperties> GetSearchServiceFromAzureAsync(AzureCloudServiceOrSearchDetails serviceDetails, ITestOutputHelper logger)
         {
-            logger.WriteLine($"Extracting search service properties from Azure. " +
-                   $"Subscription: {serviceDetails.Subscription}, " +
-                   $"Resource group: {serviceDetails.ResourceGroup}, " +
-                   $"Service name: {serviceDetails.Name}");
-
-            string result = await _azureManagementAPIWrapper.GetCloudServicePropertiesAsync(
-                                serviceDetails.Subscription,
-                                serviceDetails.ResourceGroup,
-                                serviceDetails.Name,
-                                serviceDetails.Slot,
-                                logger,
-                                CancellationToken.None);
-
-            var cloudService = AzureHelper.ParseCloudServiceProperties(result);
-
-            return new SearchServiceProperties(ClientHelper.ConvertToHttpsAndClean(cloudService.Uri), cloudService.InstanceCount);
+            if (serviceDetails.UseAzureSearchService)
```

Comment:
same as above, "AzureSearch" seems like an implementation detail. The distinction here seems to be "URLs in config" vs. "urls from Azure API"